### PR TITLE
Added issue template

### PR DESCRIPTION
**Pull request by CyberDrudge**

### CODE REVIEW

**Comment by jayvdb** [comment]
Please fix the errors the bot has mentioned

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Hello! If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on chat.
+
+### Overview Description
+
+# Steps to Reproduce
+
+1.
```
[the remark this answers is not in the document]
Comment (CyberDrudge) [reply ISSUE_TEMPLATE.md]: @jayvdb Can you help with this? What am i supposed to do here?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+2.
+3.
+
+# Actual Results
+
```
[the remark this answers is not in the document]
Comment (CyberDrudge) [reply ISSUE_TEMPLATE.md:17]: @Monal5031 can u help me fix some issues here

**Comment by Monal5031** [comment]
@CyberDrudge 
1. never ever make PRs from master.
2. Squash your commits first so we can trace problem exactly. Currently gitmate is raising problems on different commits.

**Pull request closed without merging**